utils/build-presets.ini

```diff
@@ -2365,6 +2365,29 @@ mixin-preset=stdlib_DA_standalone,build
 test
 validation-test
 
+[preset: stdlib_MSR_standalone_minimal,build]
+mixin-preset=stdlib_base_standalone
+
+build-subdir=stdlib_MSR_standalone
+min-size-release
+no-assertions
+
+verbose-build
+
+enable-experimental-differentiable-programming=0
+enable-experimental-concurrency=0
+build-swift-dynamic-sdk-overlay=0
+build-swift-dynamic-stdlib=0
+build-swift-static-stdlib=1
+stdlib-deployment-targets=generic-x86_64
+extra-cmake-options=
+    -DSWIFT_SDKS:STRING=GENERIC
+    -DSWIFT_PRIMARY_VARIANT_SDK:STRING=GENERIC
+    -DSWIFT_PRIMARY_VARIANT_ARCH:STRING=x86_64
+    -DSWIFT_GENERIC_SDK:STRING=macosx
```

Comment:
I would rather us not use extra-cmake-options. Are there build-script options that you can use here? If not, can we add them?

Reply:
Yes, but also let me explain here why each of these is needed:
- `SWIFT_SDKS=GENERIC` ... When building Darwin, the build insists to always build the macOS platform, SWIFT_SDKS will default to "GENERIC;OSX". This line prevents that.
- `SWIFT_PRIMARY_VARIANT_SDK`, `SWIFT_PRIMARY_VARIANT_ARCH` ... On Darwin, these default to macOS+x86_64, but if we're not building the macOS platform, a bunch of targets add dependencies to a non-existent target.
- `SWIFT_GENERIC_SDK` ... This configures the "generic" platform to select which SDK to actually use when building.